### PR TITLE
Add note about Micro editor

### PR DESCRIPTION
**Pull request by tomodachi94**
[Micro](https://micro-editor.github.io) supports Justfile syntax by default.

See [@zyedidiya/micro#2523](https://github.com/zyedidia/micro/pull/2523) .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -402,6 +402,10 @@ Kakoune supports `justfile` syntax highlighting out of the box, thanks to TeddyD
 
 A syntax file for Sublime Text written by TonioGela is available in [extras/just.sublime-syntax](https://github.com/casey/just/blob/master/extras/just.sublime-syntax).
 
+### Micro
+
+Micro supports Justfile syntax highlighting out of the box.
```
Comment (casey) [inline README.md]:
```suggestion
Micro supports Justfile syntax highlighting out of the box, thanks to [tomodachi94]
(https://github.com/tomodachi94).
```

File: README.md [hunk 2]
```diff
@@ -402,6 +402,11 @@ Kakoune supports `justfile` syntax highlighting out of the box, thanks to TeddyD
 
 A syntax file for Sublime Text written by TonioGela is available in [extras/just.sublime-syntax](https://github.com/casey/just/blob/master/extras/just.sublime-syntax).
 
+### Micro
+
+Micro supports Justfile syntax highlighting out of the box, thanks to [tomodachi94]
+(https://github.com/tomodachi94).
```
Comment (casey) [inline README.md]:
```suggestion
Micro supports Justfile syntax highlighting out of the box, thanks to [tomodachi94](https://github.com/tomodachi94).
```

**Comment by tomodachi94** [comment]
@casey let's wait to merge this until Micro releases a new version, so people don't get confused when it doesn't work out-of-the-box.

**Comment by casey** [comment]
Micro released a new version, so finally merging!

**Pull request merged**